articles/postgresql/concepts-high-availability.md

```diff
@@ -16,12 +16,12 @@ The Azure Database for PostgreSQL service provides a guaranteed high level of av
 ## High availability
 The high availability (HA) model is based on built-in fail-over mechanisms when a node-level interruption occurs. A node-level interruption could occur because of a hardware failure or in response to a service deployment.
 
-At all times, changes made to a PostgreSQL occur in the context of a transaction. Changes are recorded synchronously in Azure storage when the transaction is committed. If a node-level interruption occurs, the database server automatically creates a new node and attaches data storage to the new node. Any active connections are dropped and any inflight transactions are not committed.
+At all times, changes made to an Azure Database for PostgreSQL database server occur in the context of a transaction. Changes are recorded synchronously in Azure storage when the transaction is committed. If a node-level interruption occurs, the database server automatically creates a new node and attaches data storage to the new node. Any active connections are dropped and any inflight transactions are not committed.
```

Comment:
Wasn't sure what "changes made to a PostgreSQL" was referring to, so I took a guess and added "database server". Let me know if this is correct.